client/src/components/CourseInfo.tsx

```diff
@@ -1,22 +1,33 @@
 import { ExternalLink } from 'react-feather';
-
+import { HiChartBar, HiChartPie } from 'react-icons/hi';
+import { useState } from 'react';
 import { Course } from '../model/Course';
 import { CourseTerms } from './CourseTerms';
 import { RatingInfo } from './RatingInfo';
+import { Review } from '../model/Review';
+import { countRatings, classNames } from '../lib/utils';
+import _ from 'lodash';
+
+export type map = {
+  1: number;
+  2: number;
+  3: number;
+  4: number;
+  5: number;
+};
```

Comment:
Please pick a better name for this type (also why not just use an array or tuple)?

Reply:
i just thought that it had a bit better readability, since arrays index from 0